Modix/Modules/CatModule.cs

```diff
@@ -4,175 +4,39 @@
     using System.Threading;
     using System.Threading.Tasks;
     using Discord.Commands;
-    using Newtonsoft.Json;
-    using Serilog;
+    using Modix.Services.Cat;
 
     public class CatModule : ModuleBase
     {
-        // Url for cat
-        public struct URL
-        {
-            public string file { get; set; }
-        }
-
         private static readonly HttpClient Client = new HttpClient();
 
         [Command("cat"), Summary("Gets a cat")]
         public async Task Cat(string gif = "")
         {
-            // 30 seconds to find a cat
-            var cts = new CancellationTokenSource(30000);
+            // 5 seconds to find a cat
+            var cts = new CancellationTokenSource(5000);
             var token = cts.Token;
 
+            var message = string.Empty;
+
+            var cat = new CatService();
+
             // Regular picture
-            if (gif.Equals(string.Empty))
+            if (string.IsNullOrWhiteSpace(gif))
             {
-                await GetCatPicture(token);
+                message = await cat.GetCatPicture(token);
             }
             else if (gif.Equals("gif"))
```

Comment:
Should ignore case on this, otherwise "Gif" will just get the stock help message